### PR TITLE
Add employee payrolls list

### PR DESCRIPTION
**Pull request by KyleMcMaster**
Add ability to view employee's recent payrolls

### CODE REVIEW

File: client/src/app/app.module.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ import { SharedModule } from '@shared/shared.module';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { EmployeePayrollListComponent } from './employee/employee-payroll-list/employee-payroll-list.component';
```
Comment (seangwright) [inline client/src/app/app.module.ts:21]: You should be able to import this from the alias, like the other imports

Comment (KyleMcMaster) [reply client/src/app/app.module.ts]: fixed

File: client/src/app/employee/employee-list/state/employee-list.service.ts [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
+
+import { ToastrService } from 'ngx-toastr';
+
+import { EmployeeDetail } from '@employee/employee-detail/state/employee.model';
+
 import { EmployeeCreate, EmployeeListItem } from './employee-list.model';
 import { EmployeeListStore } from './employee-list.store';
 
+import { EnvService } from 'src/app/shared/env.service';
+import { ListResponse, mapListResponseToData } from 'src/app/shared/list-response';
```
Comment (seangwright) [inline client/src/app/employee/employee-list/state/employee-list.service.ts:14]: I think these absolute imports were from the solution style `tsconfig.json` that Angular has now reverted from using.

We have the `@shared` alias so these can be changed to `@shared/env.service` and `@shared/list-response`

File: client/src/app/employee/employee-payroll-list/employee-payroll-list.component.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import { Component, Input } from '@angular/core';
+
+import { EmployeePayroll } from '@employee/employee-detail/state/employee.model';
+
+@Component({
+  selector: 'app-employee-payroll-list',
+  template: ` <h5>Recent Payrolls</h5>
```
[the remark this answers is not in the document]
Comment (seangwright) [reply client/src/app/employee/employee-payroll-list/employee-payroll-list.component.ts:7]: Ya, definitely a perfect use case here.

**Comment by seangwright** [comment]
Just 1 item, otherwise looks good to me!

File: client/src/app/employee/employee-list/state/employee-list.service.ts (hunk 2 above)
Comment (KyleMcMaster) [reply client/src/app/employee/employee-list/state/employee-list.service.ts]: fixed these across the app

**Pull request merged**